spec/units/derivatives_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'hydra/works'
```

Comment:
Add comment: Including hydra-works in order to test compatibility with IndirectContainers